Android/app/src/main/res/values/strings.xml

```diff
@@ -33,7 +33,7 @@
     <string name="Map_Type_Terrain">Terrain</string>
     <string name="Map_Type_Hybrid">Hybrid</string>
     <string name="Map_Type_None">None</string>
-
+    <string name="About">About</string>
```

Comment:
Let's keep the app version info as is.

Just one general comment about adding new strings. Since there now are Japanese and French translation files as well, this string would have had to be added to those files as well.